src/js/providers/html5.js

```diff
@@ -368,6 +369,9 @@ define([
                 _setVideotagSource();
                 _videotag.load();
             } else {
+                if (_forceStateToLoad) {
```

Comment:
@robwalch We need to check this because if we just call this, it will send 'buffer' event when we try to replay the same video. I talked to Danny, and decided that we do not want to change the behavior.